beetmoverscript/templates/mini_artifact_manifest.yml

```diff
@@ -0,0 +1,603 @@
+metadata:
+  description: Maps {{ product }} artifacts to pretty names
+  name: Beet Mover Manifest
+  owner: release@mozilla.com
+s3_bucket_paths:
+  devedition_candidates: pub/devedition/candidates/{{ version }}-candidates/build{{ build_number }}/
+  devedition_candidates_repacks: pub/devedition/candidates/{{ version }}-candidates/build{{ build_number }}/
+  fennec_candidates_repacks: pub/mobile/candidates/{{ version }}-candidates/build{{ build_number }}/
+  fennec_nightly: pub/mobile/nightly/
+  fennec_nightly_repacks: pub/mobile/nightly/
+  firefox_candidates: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+  firefox_candidates_repacks: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+  firefox_nightly: pub/firefox/nightly/
+  firefox_nightly_repacks: pub/firefox/nightly/
+  thunderbird_candidates: pub/thunderbird/candidates/{{ version }}-candidates/build{{ build_number }}/
+  thunderbird_candidates_repacks: pub/thunderbird/candidates/{{ version }}-candidates/build{{ build_number }}/
+  thunderbird_nightly: pub/thunderbird/nightly/
+  thunderbird_nightly_repacks: pub/thunderbird/nightly/
+mapping:
+  win32:
+    partials[partial].artifact_name:
```

Comment:
It looks like your current artifact keys are the simple name. I think this might work, especially for people who are already familiar with the artifacts. However, something like `target.json` or `target.txt` isn't especially descriptive, and something like `version-info-json` may help. Alternately, adding a `description` field may help, so we can say

```
target.json:
    description: a json file containing version information for releases
    default: ...
    ...
```

Another issue with having the filename match the key is that it hardcodes the extension. This might be fine for json or txt, assuming we don't ever want to change the extension. Catlee is currently changing the format for the test `zip`s to `tar.gz`, and may change the extension/format once again. Since it's all in-tree, we can do a recursive find/replace... I'm wondering if it makes things easier or harder if we do something like

```
common-tests-bundle:  # I'm not in love with this name, just the first thing that came to mind
    description: Files common to the various browser tests
    simple: target.common.tests.zip
    nightly: ...
    ...
```

instead of

```
target.common.tests.zip:
    description: ...
```

That assumes we'd have functions that know how to create and extract various extensions, so maybe it is better to replace everything. Not sure.